src/number.js

```diff
@@ -26,6 +26,33 @@ define([
 	numberFormatterFn, numberFormatProperties, numberNumberingSystem, numberParserFn,
 	numberParseProperties, numberPattern, numberSymbol, stringPad ) {
 
+function validateDigits( properties ) {
+	var minimumIntegerDigits = properties[ 2 ];
+	var minimumFractionDigits = properties[ 3 ];
+	var maximumFractionDigits = properties[ 4 ];
+
+	var minimumSignificantDigits = properties[ 5 ];
+	var maximumSignificantDigits = properties[ 6 ];
```

Comment:
Well, the examples are from different projects. I wouldn't encourage having inconsistent code.

If it helps, feel free to cherry-pick https://github.com/rxaviers/globalize/commit/26f7e1049032eae3263796c7e6ed3ef15dbc4b31.